test/aerospike_clj/client_test.clj

```diff
@@ -36,8 +37,24 @@
   (let [c (client/init-simple-aerospike-client ["localhost"] "test")]
     (is c)
     (is (= "localhost" (:cluster-name c))))
-  (is (thrown-with-msg? Exception #"setting maxCommandsInProcess>0 and maxCommandsInQueue=0 creates an unbounded delay queue"
-                        (client/init-simple-aerospike-client ["localhost"] "test" {"maxCommandsInProcess" 1}))))
+
+  (letfn [(no-password? [ex]
+            (let [conf (:conf (ex-data ex))]
+              (and conf (not (contains? conf "password")))))]
+   (try
+     (client/init-simple-aerospike-client ["localhost"] "test" {"maxCommandsInProcess" 1})
```

Comment:
can you use `(is thrown? ...`

Reply:
I don’t see how. I want to examine the exception.

Reply:
you can do `(is (thrown-with-msg? c re ...))` where c is the exception class and re is a regular expression. e.g.
```
(is (thrown-with-msg? ArithmeticException #"Divide by zero"
                      (/ 1 0)))
```

Reply:
But I want to verify the exception data doesn't contain the password.

Reply:
I might be missing something, nut I assume a strict regex will do (`#"^Devide by zero$"` is an example)

Reply:
OK, I forgot `(is (thrown-with-msg?` returns the exception.
Fixed.